include/mbgl/map/camera.hpp

```diff
@@ -37,7 +37,7 @@ struct CameraOptions {
         levels. */
     optional<double> zoom;
 
-    /** Bearing, measured in degrees from true north. Wrapped to [0, 360). */
+    /** Bearing, measured in degrees from true north. Wrapped to [0, 360]. */
```

Comment:
This is still `[0, 360)`: check https://github.com/mapbox/mapbox-gl-native/blob/master/include/mbgl/math/wrap.hpp#L8-L16.

Reply:
Ah, I wasn’t considering that this was [an intentional notation](https://en.wikipedia.org/wiki/Bracket#Mathematics) for a “half-open range”. 👍